src/ts/uimanager.ts

```diff
@@ -302,6 +316,22 @@ export class UIManager {
     resolveUiVariant(null);
   }
 
+  /**
+   * Exposes i18n.t() function
+   * @returns {I18nApi.t()}
+   */
+  static localize<V extends CustomVocabulary<Record<string, string>>>(key: keyof V) {
+    return i18n.t(key);
+  }
```

Comment:
I find it really confusing that there are `t`, `localize`, and `getLocalizedText` functions, even more so that `localize` actually calls `t`. These names should be streamlined.